tests/networkpolicy_test.go

```diff
@@ -181,19 +181,19 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("[test_id:CNV-1515] should be failed to reach vmia from vmic", func() {
+		It("[test_id:1515] should be failed to reach vmia from vmic", func() {
```

Comment:
@duyanyan can you please take a look? as for me - it seems like mistake in polarion testcase steps

Reply:
@petrkotas @dshchedr Yes, the test steps are incorrect. It may cause by the incorrect case-import scripts. I have fixed them in the steps. Thanks for bringing this up.